### PR TITLE
Ins 3313 proc unit tests

### PR DESCRIPTION
**Pull request by SergeyRadist**
**- What I did**
All procedures covered with tests
**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: ledger/light/proc/send_filament.go [hunk 1]
```diff
@@ -82,3 +82,21 @@ func (p *SendFilament) Proceed(ctx context.Context) error {
 	p.dep.sender.Reply(ctx, p.message, msg)
 	return nil
 }
+
+func (p *SendFilament) replyError(
```
Comment (andreyromancev) [inline ledger/light/proc/send_filament.go]: This is no required. Error will be replied in `Init` handler.

File: ledger/light/proc/send_filament_test.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+		storageRecs := make([]record.CompositeFilamentRecord, 5)
+		filaments.RequestsMock.Return(storageRecs, nil)
+		expectedMsg, _ := payload.NewMessage(&payload.FilamentSegment{
+			ObjectID: obj,
+			Records:  storageRecs,
+		})
+
+		sender.ReplyMock.Inspect(func(ctx context.Context, origin payload.Meta, reply *message.Message) {
+			assert.Equal(t, expectedMsg.Payload, reply.Payload)
```
Comment (andreyromancev) [inline ledger/light/proc/send_filament_test.go]: Let's check origin here.

File: ledger/light/proc/send_filament_test.go [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+		var storageRecs []record.CompositeFilamentRecord
+		filaments.RequestsMock.Return(storageRecs, nil)
+
+		expectedError, _ := payload.NewMessage(&payload.Error{
+			Text: "requests not found",
+			Code: payload.CodeNotFound,
+		})
+		sender.ReplyMock.Inspect(func(ctx context.Context, origin payload.Meta, reply *message.Message) {
+			assert.Equal(t, expectedError.Payload, reply.Payload)
```
Comment (andreyromancev) [inline ledger/light/proc/send_filament_test.go]: Let's check origin here.

File: ledger/light/proc/send_object.go [hunk 4]
```diff
@@ -197,3 +189,21 @@ func (p *SendObject) Proceed(ctx context.Context) error {
 		return errors.Wrap(err, "failed to fetch record")
 	}
 }
+
+func (p *SendObject) replyError(
```
Comment (andreyromancev) [inline ledger/light/proc/send_object.go]: We can just return `CodedError` here.

File: ledger/light/proc/set_code_test.go [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+		jetID := gen.JetID()
+		recVirtual := record.Wrap(&record.Code{})
+		recordID := gen.ID()
+		rec := record.Material{
+			Virtual: recVirtual,
+			ID:      recordID,
+			JetID:   jetID,
+		}
+
+		writer.BeginMock.Return(func() {
```
Comment (andreyromancev) [inline ledger/light/proc/set_code_test.go]: This will automatically fail in not called.

Comment (SergeyRadist) [reply ledger/light/proc/set_code_test.go]: This is check that done() function is called, not Begin

File: ledger/light/proc/pass_state.go [hunk 6]
```diff
@@ -51,61 +56,71 @@ func (p *PassState) Dep(
...
-	origin := payload.Meta{}
-	err = origin.Unmarshal(pass.Origin)
-	if err != nil {
-		return errors.Wrap(err, "failed to decode origin message")
-	}
 
-	rec, err := p.dep.records.ForID(ctx, pass.StateID)
-	if err == object.ErrNotFound {
-		msg, err := payload.NewMessage(&payload.Error{Text: "no such state"})
+	sendError := func(text string, code uint32) error {
```
[the remark this answers is not in the document]
Comment (SergeyRadist) [reply ledger/light/proc/pass_state.go]: ok, cutted out one method

**Pull request merged**